### DISCUSSION TITLE
wiki/adminpro/03_advanced_operation/03_%E8%B7%AF%E7%94%B1%E9%85%8D%E7%BD%AE

### DISCUSSION THREAD

**Discussion by giscus** [SX-Code/blog-comments#21] [Announcements]
# wiki/adminpro/03_advanced_operation/03_%E8%B7%AF%E7%94%B1%E9%85%8D%E7%BD%AE

路由是项目中最重要的部分。本节将实现路由权限控制，动态生成菜单 目录结构 src router   modules     dashboard.js   base.js   constant.js   index.js   router-guards.js store   modules     asyncRoute.js  组件常量创建src/router/constant.js，填入以下内容：

https://sx-code.github.io/wiki/adminpro/03_advanced_operation/03_%E8%B7%AF%E7%94%B1%E9%85%8D%E7%BD%AE.html

**Comment by Intheautumn** [comment]
为啥我获取动态路由后    打开页面整个页面都会重定向   而不是在框架内
